index.html

```diff
@@ -586,17 +606,72 @@ <h2>
         with the <a>default display mode</a> (<code>browser</code>) being the
         last item in the chain.
       </p>
+      <p>
+        For more advanced usages, the <a data-link-for="WebAppManifest">
+        display_override</a> member can be used to specify a custom fallback
+        order. This member is valid IFF <a data-link-for="WebAppManifest">display</a> is also present.
+      </p>
+      <p>
+        The algorithm for determing the web app's requested display mode is:
+        <ol>
+          <li>[=list/For each=] <var>considering_display_mode</var> of <var>
+            <a data-link-for="WebAppManifest">display_override</a></var>:
+            <ol>
+              <li>
+                If the user agent supports the |considering_display_mode|,
+                then that display mode is used (exit).
+              </li>
+            </ol>
+          </li>
+          <li>
+            Let |considering_display_mode| be the the value specified by the
+            <a data-link-for="WebAppManifest">display</a> property.
+          </li>
+          <li>
+            If the user agent supports the |considering_display_mode|, then
+            that display mode is used (exit).
+          </li>
+          <li>
+            While |considering_display_mode| has <a>fallback display mode</a>,
+            assign that to |considering_display_mode| and go back to step 3.
+          </li>
+        </ol>
+        This algorithm will end with the final <a>fallback display mode</a> of
+        <code>browser</code>, which the user agent MUST support.
+      </p>
+      </p>
       <div class="example">
         <p>
-          For example, SuperSecure Browser (a fictitious browser) only supports
+          Example: SuperSecure Browser (a fictitious browser) only supports
           the <code>minimal-ui</code> and <code>browser</code> display modes,
           but a developer declares that she wants <code>fullscreen</code> in
-          the manifest. In this case, the user agent will first check if it
+          the manifest by using the <a data-link-for="WebAppManifest">display
+          </a> property. In this case, the user agent will first check if it
           supports <code>fullscreen</code> (it doesn't), so it falls back to
           <code>standalone</code> (which it also doesn't support), and
           ultimately falls back to <code>minimal-ui</code>.
         </p>
       </div>
+      <div class="example">
+        <p>
+          Example: Same browser as above (SuperSecure Browser),
+          but the developer instead specified the following manifest using
+          display_override:
+          <pre class="example json" title="Using display_override">
+            {
+              display_override: ["fullscreen"],
+              display: "browser"
+            }
+          </pre>
+          This declares that she only wants <code>fullscreen</code>, and if
```

Comment:
Avoid using gendered pronouns (in general, best to avoid talking about the developer and just the relationship between the site and the user agent, which is what this spec cares about). Just "This declares that the user agent should use fullscreen, and if that is not supported, it should fall back to browser without considering the other modes."